participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java

```diff
@@ -8,7 +8,22 @@
 
 package com.google.cloud.healthcare.fdamystudies.config;
 
+import com.google.cloud.storage.Storage;
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed review comment.